README.md

```diff
@@ -88,20 +88,20 @@ Learn more about using KaTeX [on the website](https://katex.org)!
 
 ### Code Contributors
 
-This project exists thanks to all the people who contribute. [[Contribute](CONTRIBUTING.md)].
+This project exists thanks to all the people who contribute code. If you'd like to help, see [our guide to contributing code](CONTRIBUTING.md).
 <a href="https://github.com/KaTeX/KaTeX/graphs/contributors"><img src="https://opencollective.com/katex/contributors.svg?width=890&button=false" /></a>
 
 ### Financial Contributors
 
-Become a financial contributor and help us sustain our community. [[Contribute](https://opencollective.com/katex/contribute)]
```

Comment:
Maybe add the link to "Become a financial contributor"?

Reply:
The last image will show as a button.

Reply:
For accessibility, we should add an `alt` attribute to said button, or add the link to the text.